<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>profile_select</class>
 <widget class="QWidget" name="profile_select">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#profile_select, ProfileSelect {
	background-color: #2A2A2A;
}

#frame {
	background-color: #252525;
	border-radius: 16px;
}

#scroll_profiles, #scrollcontent_profiles {
	background-color: #252525;
}

#label {
	font-size: 18pt;
	font-weight: 700;
}

#btn_create:hover, #btn_create:focus {
	background-color: #353535;
	border-radius: 6px;
}

#icon_create {
	margin: 16px;
}</string>
  </property>
  <layout class="QVBoxLayout" name="layout_widget">
   <item alignment="Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignVCenter">
    <widget class="QFrame" name="frame">
     <property name="minimumSize">
      <size>
       <width>700</width>
       <height>300</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>700</width>
       <height>300</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="layout_frame">
      <property name="spacing">
       <number>20</number>
      </property>
      <property name="leftMargin">
       <number>40</number>
      </property>
      <property name="topMargin">
       <number>40</number>
      </property>
      <property name="rightMargin">
       <number>40</number>
      </property>
      <property name="bottomMargin">
       <number>40</number>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Select profile</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QScrollArea" name="scroll_profiles">
        <property name="frameShape">
         <enum>QFrame::Shape::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Shadow::Plain</enum>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
        </property>
        <widget class="QWidget" name="scrollcontent_profiles">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>620</width>
           <height>168</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="layout_profiles">
          <item>
           <spacer name="spacer_left">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="FrameButton" name="btn_create">
            <property name="maximumSize">
             <size>
              <width>108</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="focusPolicy">
             <enum>Qt::FocusPolicy::StrongFocus</enum>
            </property>
            <layout class="QVBoxLayout" name="layout_create">
             <property name="leftMargin">
              <number>6</number>
             </property>
             <property name="topMargin">
              <number>6</number>
             </property>
             <property name="rightMargin">
              <number>6</number>
             </property>
             <property name="bottomMargin">
              <number>6</number>
             </property>
             <item>
              <widget class="IconWidget" name="icon_create">
               <property name="minimumSize">
                <size>
                 <width>96</width>
                 <height>96</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>96</width>
                 <height>96</height>
                </size>
               </property>
               <property name="text">
                <string notr="true"/>
               </property>
               <property name="pixmap">
                <pixmap resource="../../../resources/index.qrc">:/icons/plus</pixmap>
               </property>
               <property name="scaledContents">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_create">
               <property name="text">
                <string>New Profile</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignmentFlag::AlignCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <spacer name="spacer_right">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FrameButton</class>
   <extends>QFrame</extends>
   <header>widgets.common</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>IconWidget</class>
   <extends>QLabel</extends>
   <header>widgets.common</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>scroll_profiles</tabstop>
  <tabstop>btn_create</tabstop>
 </tabstops>
 <resources>
  <include location="../../../resources/index.qrc"/>
  <include location="../../../resources/index.qrc"/>
 </resources>
 <connections/>
</ui>
